### ISSUE TITLE
iPadOS backend

### ISSUE THREAD

**Issue by w1ebr** [ocornut/imgui#6924]
Has anyone worked on an iOS / iPadOS backend? There is a "metal on OSX" example that includes a backend and works well, and a "metal on iOS" example without a backend that doesn't work very well. I don't have enough experience to create a new backend myself.

**Comment by ocornut** [comment]
It is the same backend used in both examples.
You may use SDL instead if it makes things easier for you.

**Comment by w1ebr** [comment]
No, there is an extra file called imgui_metal_osx for the osx case and no equivalent file for ios.I'll look at sdl,thanks.On Oct 14, 2023 11:42 AM, omar ***@***.***> wrote:
It is the same backend used in both examples.
You may use SDL instead if it makes things easier for you.

**Comment by thedmd** [comment]
`example_apple_metal` for iOS does feed ImGui directly, which translate touch input to mouse movements.
There is no `imgui_impl_ios` backend present.
Backends at this point are usually contributed by the users. I think one for iOS woudn't have a hard time to pass PR. 

As suggested by Omar, standalone iOS app can be built with SDL as a backend.
